packages/@aws-cdk/aws-lambda-destinations/test/integ.lambda-chain.ts

```diff
@@ -0,0 +1,36 @@
+import * as lambda from '@aws-cdk/aws-lambda';
+import { App, Construct, Stack, StackProps } from '@aws-cdk/core';
+import * as destinations from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const lambdaProps: lambda.FunctionProps = {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      handler: 'index.handler',
+      code: lambda.Code.fromInline(`exports.handler = async (event) => {
+        console.log(event);
+        return event;
+      };`),
+    };
+
+    const first = new lambda.Function(this, 'First', lambdaProps);
+    const second = new lambda.Function(this, 'Second', lambdaProps);
+    const third = new lambda.Function(this, 'Third', lambdaProps);
+
+    first.configureAsyncInvoke({
+      onSuccess: new destinations.LambdaDestination(second, { responseOnly: true }),
+    });
+
+    second.configureAsyncInvoke({
+      onSuccess: new destinations.LambdaDestination(third, { responseOnly: true }),
+    });
+  }
+}
+
+const app = new App();
```

Comment:
Can we also configure an `onFailure()` as part of this integ test, just to get as much coverage?

It would be great if we could wire it up in a way where a couple of AWS CLI calls can validate that the lambda functions actually work as set up. Something like [this](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer.ts#L6-L9) and [this](https://github.com/aws/aws-cdk/pull/5710/files#diff-0a7f68516bc170fd3cbd24db47d60e2d).

Not strictly necessary but would be a good way for someone working on this piece of code as validation.

Reply:
No problem, this is similar to what I already did for `integ.destinations.ts`.